renderer-macos/lib/bindings/BriskMenu.re

```diff
@@ -45,66 +46,20 @@ type action =
   /* Custom callback */
   | Callback(unit => unit);
 
-type target =
```

Comment:
I just realized that we don't need anything but NSApp target as menu actions should be app-wide. Is that the case here?

Reply:
All actions that are not `Callback` should have NSApp as the target. Specifiying it separately is a bit extra.